openshift_metrics/utils.py

```diff
@@ -223,14 +221,17 @@ def write_metrics_by_namespace(
     rows = []
     namespace_annotations = get_namespace_annotations()
     headers = [
-        "Namespace",
-        "Coldfront_PI Name",
-        "Report Start Date",
-        "Report End Date",
-        "_cpu_hours",
-        "_memory_hours",
-        "SU TYPE",
-        "SU Type Hours",
+        "Invoice Month",
+        "Project - Allocation",
+        "Project - Allocation ID",
+        "Manager (PI)",
+        "Invoice Address",
+        "Institution",
+        "Institution - Specific Code",
+        "SU Hours (GBhr or SUhr)",
+        "SU Type",
+        "Rate",
```

Comment:
@joachimweyl I noticed that the google doc calls this "Invoicing Rate". Just want to double check before I update this.